src/app/case/case-feature/container/create-case/create-case.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+
+//import { LoggerService } from '../../../../logger/logger.service'
+
+@Component({
+  selector: 'app-create-case',
+  templateUrl: './create-case.component.html',
+  styleUrls: ['./create-case.component.css']
+})
+export class CreateCaseComponent implements OnInit {
+
+  caseSelected: string
+  constructor(
+    //private logger: LoggerService
+  ) {
+  }
+
+  ngOnInit() {
+
+    //this.logger.info('CreateCaseComponent: initialized');
```

Comment:
we can remove this if it is commented out.

Reply:
done